src/libs/actions/IOU.js

```diff
@@ -562,8 +562,11 @@ function createDistanceRequest(report, participant, comment, created, transactio
  * @param {Object} [receipt]
  */
 function requestMoney(report, amount, currency, created, merchant, payeeEmail, payeeAccountID, participant, comment, receipt = undefined) {
+    // If the report is iou or expense report, we should get the chat report of this to pass to getMoneyRequestInformation function
+    const isMoneyRequestReport = ReportUtils.isMoneyRequestReport(report);
+    const currentReport = isMoneyRequestReport ? ReportUtils.getReport(report.chatReportID) : report;
```

Comment:
I think it would be better to call it like this: 
```suggestion
    const chatReport = isMoneyRequestReport ? ReportUtils.getReport(report.chatReportID) : report;
```

Reply:
currentChatReport